src/main/java/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -122,18 +130,30 @@ public static DateFields getFields(Date d, String timezone) {
         cd.setTime(d);
         if (timezone != null) {
             cd.setTimeZone(TimeZone.getTimeZone(timezone));
+        } else if (timezoneOffset() != -1) {
```

Comment:
Ah I see, we check for `-1` here

Reply:
This sounds like there was a comment you made before it that didn't go through? Did you have a concern here?

Reply:
Was in reference to https://github.com/dimagi/formplayer/pull/585/files#r160769302